.github/workflows/release.yaml

```diff
@@ -214,6 +214,7 @@ jobs:
             ./build/*.rpm
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CODER_GPG_RELEASE_KEY_BASE64: ${{ secrets.CODER_GPG_RELEASE_KEY_BASE64 }}
```

Comment:
Just curious, why base64 vs armored?

Reply:
It's armored *and* base64'd. Base64'd mostly so it's on a single line. It's not necessary, but avoids any weird escaping issues if they ever crop up in the future.